src/soilbiogeochem/SoilBiogeochemPotentialMod.F90

```diff
@@ -17,6 +17,8 @@ module SoilBiogeochemPotentialMod
   use SoilBiogeochemNitrogenStateType    , only : soilbiogeochem_nitrogenstate_type
```

Comment:
Looks like these changes can be dropped.

Reply:
Two empty comment lines were removed. Do we need keep it?